securedrop/tests/test_journalist.py

```diff
@@ -67,44 +78,26 @@ def test_col_un_star_call_db(self, db_session):
 
         db_session.commit.assert_called_with()
 
-    def set_up_request(self, cols_selected, action):
-        journalist.request.form.__contains__.return_value = True
-        journalist.request.form.getlist = MagicMock(return_value=cols_selected)
-        journalist.request.form.__getitem__.return_value = action
 
     @classmethod
     def tearDownClass(cls):
         # Reset the module variables that were changed to mocks so we don't
         # break other tests
         reload(journalist)
 
-
+seshy = None
```

Comment:
Maybe it got accidentally pasted in there since it's out of context. Anyway, removed now.